portfolio/src/main/webapp/misc.html

```diff
@@ -12,5 +13,18 @@
       <a href="socials.html">socials</a>
       <a class="active" href="misc.html">misc</a>
     </div>
+    <div class="content">
+      <h2>travel</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam volutpat justo sit amet accumsan pellentesque.
+        Praesent placerat varius erat a accumsan.</p>
+      <div class="image-grid-row">
```

Comment:
Did you want to center this div? div.content is a fixed width, and these images don't take it all up, so it's slightly left-aligned.